### ISSUE TITLE
Flutter web - Is it possible not to save the screen browser history?

### ISSUE THREAD

**Issue by RaineCode178** [Milad-Akarie/auto_route_library#522]
I'm using flutter web and I have this 2 screen. I don't want to save a history of that 1 screen in the browser history so if I pressed the back button in the browser it wont go back, back to the previous browser history. I'm using the `navigate() ` method

**Comment by theweiweiway** [comment]
correct me if im wrong @Milad-Akarie but would `.replace` work? That would not push the 1st screen into the stack right

**Comment by Milad-Akarie** [comment]
@theweiweiway I think what he's talking about is removing an entry from the browser's history which can not be done at the moment, I talked about this with @chunhtai the maintainer of Navigator 2.0 and he acknowledged the issue, so I guess we just have to wait for the fix. 

@RaineCode178  there's something that might work depending on your case
let's say you're at 
`/page1`
if you push  `/page2` inside of this special method the url won't change it will still say `/page1`
```dart
Router.neglect(contex,(){
   contex.router.push(Page2Route());
})
```

**Comment by theweiweiway** [comment]
very very interesting :O

**Comment by RaineCode178** [comment]
@Milad-Akarie Yeah thats im trying to do removing an entry from the browser's history. Because before on navigator 1.0, I used the `pushNamedAndRemoveUntil()` to removed a screen widget on the stack tree. Now since its in the browser history I can keep navigating back and forth to that screen and I dont want that to happen on my case.

I tried your example given and its working fine. Ill just use this method for now until the update you said will come. Thank you
